controllers/configmap_controller.go

```diff
@@ -85,7 +86,15 @@ func NewConfigMapReconciler(mgr manager.Manager, clusterConfig cluster.Config, w
 	if err != nil {
 		return nil, errors.Wrap(err, "unable to initialize Prometheus configuration")
 	}
-	svcData, err := servicescm.NewData(&[]servicescm.Service{}, &[]servicescm.FileInfo{})
+	svcData, err := servicescm.NewData(&[]servicescm.Service{{
+		Name:                         windows.WindowsExporterServiceName,
+		Command:                      windows.WindowsExporterServiceCommand,
+		NodeVariablesInCommand:       nil,
+		PowershellVariablesInCommand: nil,
+		Dependencies:                 nil,
+		Bootstrap:                    false,
+		Priority:                     1,
+	}}, &[]servicescm.FileInfo{})
```

Comment:
Consider introducing 2 constants to hold the `managedServices` and `managedFiles`. Doing this in-line may get messy as this continues to grow

Reply:
Fair enough, how about separate variables then? There will be services managed by WICD across all cluster configs.

Reply:
Done. Additionally I expect this logic to be broken out into a separate function once the generation gets more complicated.